docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -0,0 +1,103 @@
+# Deprecating, Disabling, and Removing Formulae
+
+There are many reasons why formulae may be deprecated, disabled, or removed. This document explains the differences between each method as well as explaining when one method should be used over another.
+
+## Overview
+
+This general rule of thumb can be followed:
+
+- `deprecate!` should be used for formulae that _should_ no longer be used.
+- `disable!` should be used for formulae that _cannot_ be used.
+- Formulae that are no longer acceptable in homebrew/core or have been disabled for over a year should be removed.
+
+## Deprecation
+
+If a user attempts to install a deprecated formula, they will be shown a warning message but the install will succeed.
```

Comment:
```suggestion
If a user attempts to install a deprecated formula, they will be shown a warning message but the install will proceed.
```
Minor nitpick; feel free to ignore. I just think that there could be other reasons why the install will fail, even if a formula isn't disabled.

Reply:
Makes sense to me